### PR TITLE
Update azure-stack-add-scale-node.md

### PR DESCRIPTION
**Pull request by rammalkani**
We need to note that expansion is possible, but scale in operations are unsupported. This page appears to be best to add the note.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@rammalkani : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattbriggs — CHANGES_REQUESTED** [review]
**Current**: The current architecture of Azure Stack Hub delivering Azure consistent services does not support removing scale unit nodes for the purpose of scaling down, only expansion of capacity through the add node process is possible.   

**Suggestion**: Azure Stack Hub doesn't support removing scale unit nodes for the purpose of scaling down. It is only possible to expand capacity by the addition of nodes.

Hi ... can you rephrase to something like the suggestion. The consistent message seems like a non sequitur in this context.

**Comment by mattbriggs** [comment]
@rammalkani I have a small language change request. Thanks! Matt

#in-progress

**Comment by PRMerger15** [comment]
@rammalkani : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rammalkani** [comment]
Made the language change as suggested.

**Comment by mattbriggs** [comment]
#sign-off
